services/slam/fake/slam_test.go

```diff
@@ -170,3 +240,88 @@ func reverse[T any](slice []T) []T {
 	}
 	return slice
 }
+
+func verifyGetPointCloudMapStreamStateful(t *testing.T, mimeType string, slamSvc *SLAM, extra map[string]interface{}) {
+	testDataCount := maxDataCount
+	getMapPcdResults := []float64{}
+	positionResults := []spatialmath.Pose{}
+	getPositionResults := []spatialmath.Pose{}
+	getInternalStateResults := []int{}
+
+	// Call GetPointCloudMapStream twice for every testData artifact
+	for i := 0; i < testDataCount*2; i++ {
+		pInFrame, err := slamSvc.Position(context.Background(), slamSvc.Name, extra)
```

Comment:
[q] do we need additional testing for old endpoint when they will be removed in the next week (non-blocking question)

Reply:
I figured it didn't take much to leave them in & they can be removed when we remove the old endpoints.